prometheus-formula/prometheus-formula.changes

```diff
@@ -1,4 +1,9 @@
+-------------------------------------------------------------------
+Tue Feb  8 13:46:50 UTC 2022 - Witek Bedyk <witold.bedyk@suse.com>
+
+- Version 0.6.0
   * Add support for new Uyuni SD in Prometheus >= 2.31
+  * Fix Blackbox exporter configuration for Prometheus >= 2.31
```

Comment:
0.6.0 was never submitted to any codestream, so LGTM.

Change codes are on @renner